doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,495 @@
+        * Example: 1 per dpu interface
+
+* Mac allocation per DPU
+    * default = 8
+        * Example: management port, dockers, loopback, 1 for midplane-interface dpu end point, plus some spare
+    * application extension = 8
+        * Note: 8 mac addresses for future application expansion
+ 
+* Total mac for SmartSwich = (512 + 8 + 8) + ((8 + 8) * 8) =   656 mac addresses
+
+* The MAC address for each host endpoint and the corresponding DPU endpoint will be read from the hardware and updated into the MID_PLANE_IP_MAC table in the ChassisStateDB as shown below. The IP address will also be stored here for convenience.
+
+### 3.3.2.  ChassisStateDB Schema for MID_PLANE_IP_MAC
+```
+Table: “MID_PLANE_IP_MAC”
+
+Key: "midplane_interface|dpu0"
+            "id”: “1”,
+            "host_ip": “169.254.1.2”,
+            “host_mac”: “BA:CE:AD:D0:C0:01”, # mac is an example
+            "dpu_ip": “169.254.1.1”,
+            “dpu_mac”: “BA:CE:AD:D0:D0:01”  # will be updated by the DPU
+```
+
+## 3.4. Debuggability & RMA
+CLI Extensions and Additions
+
+show platform inventory - shows the DPUs on the switch
+```
+root@sonic:~#show platform inventory
+
+    Name                Product ID      Version         Serial Number   Description
+
+Chassis
+    CHASSIS             28FH-DPU-O 	0.10            FLM274802ER     28x400G QSFPDD DPU-Enabled 2RU Smart Switch,Open SW
+
+Route Processors
+    RP0                 28FH-DPU-O 	0.10            FLM274802ER     28x400G QSFPDD DPU-Enabled 2RU Smart Switch,Open SW
+
+DPU Modules
+    DPU0                8K-DPU400-2A    0.10            FLM2750036X     400G DPU 
+    DPU1                8K-DPU400-2A    0.10            FLM2750036S     400G DPU 
+    DPU2                8K-DPU400-2A    0.10            FLM274801EY     400G DPU 
+    DPU3                8K-DPU400-2A    0.10            FLM27500371     400G DPU
+
+Power Supplies                                                                
+    psutray                                                                                                                                                             
+        PSU0            PSUXKW-ACPI     0.0             POG2427K01K     AC Power Module with Port-side Air Intake                                                 
+        PSU1            PSUXKW-ACPI     0.0             POG2427K00Y     AC Power Module with Port-side Air Intake                                                 
+
+Cooling Devices
+    fantray0            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray1            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray2            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray3            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan
+
+FPDs
+    RP0/info.0                          0.5.6-253      
+
+```
+
+show platform temperature - shows the DPU temperature on the switch
```

Comment:
@rameshraghupathy Please indicate (preferably at the top right corner in red ink) where this CLI is executed: DPU or NPU

Reply:
Done